tools/jira_cmd.py

```diff
@@ -340,8 +340,8 @@ def main(args):
     selectors.add_argument("-nf", "--named-filter", action=buildEpicFilterAction, dest="search_query",
                            help="Search for Epics matching the given named filter")
     selectors.add_argument("-tt", "--triaging-tickets", action='store_const',
-                           dest="search_query", const='project = MGMT AND component = "Assisted-installer Triage"',
-                           help="Search for Assisted Installer triaging tickets")
+                           dest="search_query", const='project = MGMT AND component = "Assisted-installer Triage" AND resolved > -7d',
```

Comment:
'-tt' is meant to return all the triaging tickets.

You can either add a new param for this, or just use the '-s' param and specify the query

Reply:
ok. missed the a row below with the -rtt
I will change it to use resolved instead of created